evc-sumo/docker/build-image.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+#  Copyright (C) 2023 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+USERNAME=usdotfhwastol
+
+cd "$(dirname "$0")"
+# IMAGE=$(./get-image-name.sh | tr '[:upper:]' '[:lower:]')
+IMAGE=evc
```

Comment:
EVC is a little non-descriptive of a name to me, I know it means things to us internally but it wouldn't mean much to an external user without us explaining it. Maybe just spelling it out in full `econolite-virtual-controller`?

Reply:
changed name to suggested econolite-virtual-controller